### PR TITLE
doc updates

### PR DESCRIPTION
**Pull request by eschultink**
### Fixes
 - fix FAQ on VPC/API gateway
 - cleanup some versioning in jira docs/etc

### Change implications

 - dependencies added/changed? **no**
 - something important to note in future release notes **no**

### CODE REVIEW

**Review by davidfq — APPROVED** [review]
👍 I don't see any structure changes that could impact `SUMMARY` in #702

File: docs/faq-security.md [hunk 1]
```diff
@@ -62,23 +65,19 @@ Worklytics that is otherwise not exposed to the internet.
...
+VPC usage *requires* an API Gateway to be deployed in front of the proxy instances.
 
-The usual cases for deploying infra in a VPC is to isolate a complex, potentially insecure
-component, such as a VM with lots of packages/software/etc running on it. Proxy instances are small
-java bundles, compiled, packaged, and deployed by your team - into serverless infrastructure
-sandboxes (AWS Lambda, GCP CLoud Functions, etc). The code is source-available for your team to
-review; it undergoes full automated testing and continual vulnerability scanning.
+Please note that proxy instances generally use the public APIs of cloud SaaS tools, so do not require
+access to your internal network/VPN unless you are connecting to an on-prem tool (eg, GitHub
+Enterprise Server, etc).  So there is no technical reason to deploy Psoxy instances in a VPC.
```
Comment (davidfq) [inline docs/faq-security.md:74]: Should we highlight this section?

File: docs/aws/guides/lambdas-on-vpc.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Lambdas on a VPC
 
-**alpha** - this is a work in progress, and unsupported; may change in backwards incompatible ways.
+**beta** - This is now available for customer-use, but may still changein backwards incompatible
```
Comment (davidfq) [inline docs/aws/guides/lambdas-on-vpc.md:3]:
```suggestion
**beta** - This is now available for customer-use, but may still change in backwards incompatible
```

File: docs/faq-security.md [hunk 3]
```diff
@@ -62,23 +65,19 @@ Worklytics that is otherwise not exposed to the internet.
 
 ## Can I deploy Psoxy instances in a VPC?
 
-As of November 2023, that is not directly supported by the Worklytics-provided Terraform modules. If
-you have a use-case that you believe requires a VPC, please let us know.
+VPC support is available as a *beta* feature as of February 2024.
```
Comment (davidfq) [inline docs/faq-security.md:68]: Reason for this date?

**Review by aperez-worklytics — APPROVED** [review]

File: docs/faq-security.md (hunk 3 above)
Comment (eschultink) [reply docs/faq-security.md:68]: just informational ... we used to tell people it wasn't supported, but now we've done it.

**Pull request merged**